src/os/BtrfsFileStoreBackend.cc

```diff
@@ -540,7 +543,7 @@ int BtrfsFileStoreBackend::clone_range(int from, int to, uint64_t srcoff, uint64
   a.dest_offset = dstoffclone;
   err = ::ioctl(to, BTRFS_IOC_CLONE_RANGE, &a);
   if (err >= 0) {
-    r += err;
+    r += 0; //expect this.It don't do anycopy in kernel.
```

Comment:
In fact, i modify the meaning of return value for this function. So i use 'r += 0' and a comment to note this.